peachjam/signals.py

```diff
@@ -55,3 +65,22 @@ def extracted_citation_deleted(sender, instance, **kwargs):
     """Update citation counts on works."""
     ExtractedCitation.update_counts_for_work(instance.citing_work)
     ExtractedCitation.update_counts_for_work(instance.target_work)
+
+
+@receiver(signals.post_save, sender=User)
+def add_saved_document_permissions(sender, instance, created, **kwargs):
+    if pj_settings().allow_save_documents and created:
+        permissions = Permission.objects.filter(
```

Comment:
It's much better to do this with a group, because then the perms can be added or removed on a group basis. Otherwise if we want to disable this permission, we have to go through each user and remove the perm. So we'll need to create a default group (perhaps "All users", it can be hardcoded) and then ensure all new users belong to that group. It's probably easiest to get_or_create the Group object so that if it's deleted, it's always re-created.